### DISCUSSION TITLE
Video Player going into fullscreen mode automatically when playing

### DISCUSSION THREAD

**Discussion by murphypetercl** [vidstack/player#1472] [Q&A]
Hi, on one of my users device the video player is going into fullscreen mode automatically when user plays the video. He is using a HP ProBook. I am not able to reproduce on my Dell Latitude. I am using vue media-player component and media-video-layout. I have not configured anything in relation to fullscreen mode. 
Please advise if there is any reason why this would happen on certain devices and how can I prevent this from happening? It should only go to fullscreen mode if user explicitly selects fullscreen mode in the player.
Thanks.

**Comment by rossanodr** [comment]
set  playsInline to true
